src/org/pentaho/di/ui/job/entries/sqoop/AbstractSqoopJobEntryController.java

```diff
@@ -81,12 +83,11 @@
   // The following is overwritten in init() with the i18n string
   protected DatabaseItem USE_ADVANCED_OPTIONS = new DatabaseItem( "@@advanced@@", "Use Advanced Options" );
 
-  private NamedCluster CHOOSE_AVAILABLE_CLUSTER;
+  //private NamedCluster CHOOSE_AVAILABLE_CLUSTER;
```

Comment:
@e-cuellar is there any reason to keep this line of code alive in commented form?